modules/nw-multus-ipam-object.adoc

```diff
@@ -310,4 +310,62 @@ pod/whereabouts-reconciler-p4sxw 1/1 Running 0 6s
 pod/whereabouts-reconciler-rvfdv 1/1 Running 0 6s
 pod/whereabouts-reconciler-svzw9 1/1 Running 0 6s
 daemonset.apps/whereabouts-reconciler 6 6 6 6 6 kubernetes.io/os=linux 6s
+----
+
+[id="nw-multus-creating-whereabouts-dualstack-config_{context}"]
+== Creating a configuration for assignment of dualstack IP addresses dynamically
+
+DualStack IP address assignment can be configured with the `ipRanges` parameter for:
+- IPv4 addresses
+- IPv6 addresses
+- multiple IP address assignment 
+
+.Procedure
+
+. Set type to whereabouts
+. Use `ipRanges` to allocate IP addresses.
+
+For example, defining the range as `192.168.10.1/24` excludes the first network address and the last broadcast address.
```

Comment:
I think we can omit this line? I think we cover setting a CIDR range elsewhere in the Whereabouts docs.

Reply:
It seemed like it may not have been totally relevant but I leave that to your discretion